godot-core/src/registry/plugin.rs

```diff
@@ -96,6 +96,8 @@ pub enum PluginItem {
 
         /// Whether the class has a default constructor.
         is_instantiable: bool,
+        /// Base, Description, Members.
+        docs: [&'static str; 3],
```

Comment:
Personally i think it'd be nicer to have a custom struct here, that way we can name each of these, rather than needing to rely on documenting what each of the three string are.